src/Modules/Admin/Commands.Admin/help/Get-PowerBIActivityEvents.md

```diff
@@ -0,0 +1,115 @@
+---
+external help file: Microsoft.PowerBI.Commands.Admin.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Admin
+online version:
+schema: 2.0.0
+---
+
+# Get-PowerBIActivityEvents
+
+## SYNOPSIS
+Retrieves the audit activity events for the tenant.
+
+## SYNTAX
+
+```
+Get-PowerBIActivityEvents -StartDateTime <String> -EndDateTime <String> [-ActivityType <String>]
+ [-ResultType <Int32>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Retrieves the audit activity events by the tenant administrator for the organization.
+
+This cmdlet requires the calling user to be a tenant administrator of the Power BI service.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-PowerBIActivityEvents -StartDateTime 2019-08-10T14:35:20 -EndDateTime 2019-08-10T18:25:50
+```
+
+### Example 2
+```powershell
+PS C:\> Get-PowerBIActivityEvents -StartDateTime 2019-08-10T14:35:20 -EndDateTime 2019-08-10T18:25:50 -ActivityType viewreport -ResultType 1
+```
+
+## PARAMETERS
+
+### -ActivityType
+Filters the results based on a boolean condition. Currently only supporting filterting based on audit event record's 'Activity'
```

Comment:
Filters activity records that have this activity type applied.

Something along those lines, the current description is too implementation focused instead of consumer focused